fuse_ll/src/fuse/mount.rs

```diff
@@ -49,8 +153,12 @@ mod param {
     pub const MFSTYPENAMELEN: size_t = 16; // length of fs type name including null
     pub const MAXPATHLEN: size_t = 1024; //PATH_MAX
 
+    use super::common_param::*;
+    pub const VALID_ON_OFF_OPTIONS: [&str; 3] = [ALLOW_OTHER, RW, RO];
```

Comment:
this is hard to maintain, what about use a function to define these options?

fn get_on_off_options() -> Vec<&str> {
    vec!["allow_other", "rw", "ro", ...]
}